htdocs/core/modules/import/import_csv.modules.php

```diff
@@ -679,7 +680,7 @@ public function import_insert($arrayrecord, $array_match_file_to_database, $obji
 									}
 
 									// Load content of field@table into cache array
-									if (!is_array($this->cachefieldtable[$cachekey])) { // If content of field@table not already loaded into cache
+									if (isset($this->cachefieldtable[$cachekey]) && !is_array($this->cachefieldtable[$cachekey])) { // If content of field@table not already loaded into cache
```

Comment:
If cachefieldtable[cachekey]  was unset, test was true + warning before the pr, after it seems wrong + no warning. Is this behaviour change done on purpose ?

Reply:
@eldy i send a fix